src/themes/teams/components/List/listItemStyles.ts

```diff
@@ -49,6 +49,7 @@ const listItemStyles: ComponentSlotStylesInput<ListItemProps, any> = {
     }
   },
   header: ({ variables }): ICSSInJSStyle => ({
+    color: variables.headerColor,
```

Comment:
Should this be changed for all list items?

Reply:
The reason I added this is to be able to change it from Dropdown, on item highlight. And the default color was the same before this change anyway. Do you think we have a better alternative?